msg/PositionSetpoint.msg

```diff
@@ -31,6 +31,8 @@ bool loiter_direction_counter_clockwise # loiter direction is clockwise by defau
 float32 acceptance_radius   # navigation acceptance_radius if we're doing waypoint navigation
 
 float32 cruising_speed		# the generally desired cruising speed (not a hard constraint)
+float32 vertical_up_speed
+float32 vertical_down_speed
```

Comment:
Could you comment the unit and description?

E.g.
```
# [m/s] Vertical upward speed setpoint
```

Reply:
updated